### PR TITLE
New Annotation Study Results layout

### PR DESCRIPTION
**Pull request by msmolens**
TODO:
- [x] Spacing between sections
- [x] Make header appearance consistent; don't abuse 'label' for section headers
- [x] Dropdowns shouldn't fill width
- [x] Global feature table shouldn't expand to fill width
- [x] New user selection interface
- [x] Add study details
- [x] Add image count
- [x] Add image thumbnails
- [x] Fit more information on screen (reduce whitespace, possibly move study selection to a separate mode or page, reduce font sizes, etc.)
- [ ] Make layout/interaction more intuitive
- [x] Display image on "Image Preview" tab instead of "Global Features"
- [x] Hide feature tabs until user selects an image

Possible future work:
- [ ] Allow selecting multiple users (requires server support)
- [ ] Allow selecting multiple local features (requires server support)
- [ ] Add image metadata (i.e. number of annotations, raters)
- [ ] Loading indicators
- [ ] Add button to add users to study (move from Manage Annotation Studies)
- [ ] Add button to create new study (move from Manage Annotation Studies)

### CODE REVIEW

**Comment by msmolens** [comment]
@brianhelba PTAL at what's here so far.

File: web_external/templates/widgets/studyResultsStudyDetailPage.jade [hunk 1]
```diff
@@ -4,6 +4,10 @@
     - var featureset = model.get('featureset')
     table.table.table-condensed.isic-study-results-study-details-table
       tr
+        td Unique ID
```
Comment (brianhelba) [inline web_external/templates/widgets/studyResultsStudyDetailPage.jade]: I realized this should have the `.isic-study-results-study-details-table-label` class.

File: web_external/templates/widgets/studyResultsStudyDetailPage.jade [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.isic-study-results-study-details-container
+  .isic-study-results-header Study Details
+  if hasStudy
+    - var featureset = model.get('featureset')
+    table.table.table-condensed.isic-study-results-study-details-table
+      tr
+        td Unique ID
```
Comment (brianhelba) [inline web_external/templates/widgets/studyResultsStudyDetailPage.jade]: I realized this should have the `.isic-study-results-study-details-table-label` class.

**Comment by msmolens** [comment]
@brianhelba This is ready to merge

**Comment by brianhelba** [comment]
Excellent, LGTM.
Merging.

**Pull request merged**